Gulpfile.js

```diff
@@ -99,14 +92,18 @@ gulp.task("test", done =>
 {
     const lPathArgs = getArgs()["path"];
     const lFileArgs = getArgs()["file"];
+    process.env.ENV__LOGGING_LEVEL = "OFF";
+    let lArgs = [];
 
     if (lPathArgs !== undefined)
```

Comment:
```suggestion

    let lArgs = [];
    if (lPathArgs !== undefined)
```

The whole purpose of the `if else` block is to set the value of `lArgs` as such they should be grouped

Reply:
done